src/database/queries/addData.js

```diff
@@ -0,0 +1,22 @@
+const connection = require('../config/db_connection');
+
+const addUser = (userInfo) => {
+  const queryObj = {
+    text: 'insert into users (firstname, lastname, mobile_number, email, freelancer_url, photo_url,birthday, specalization_id,city,password ) values ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10) RETURNING *',
```

Comment:
I think this query return error, because of the city and the birthday null in the first :+1:  
and pls make your code more readable  like 
```javascript
const addUser = (userInfo) => {
  const { firstname, lastname, mobile_number, email, specalization_id, freelancer_url, photo_url, password } = userInfo;
  const queryObj = {
    text: 'insert into users (firstname, lastname, mobile_number, email, freelancer_url, photo_url, specalization_id,password ) values ($1, $2, $3, $4, $5, $6, $7, $8) RETURNING *',
    values: [
      firstname,
      lastname,
      mobile_number,
      email,
      freelancer_url,
      photo_url,
      specalization_id,
      password,
    ],
  };
  return connection.query(queryObj);
};
 
```